### ISSUE TITLE
Display scaling messes up mouse cursor in Gnome apps.

### ISSUE THREAD

**Issue by Dark-Thoughts** [ublue-os/bazzite#1858]
### Describe the bug

When I use higher than 100% display scaling, the mouse cursor gets absurdly big and blurry within Gnome apps such as BoxBuddy or the konsole that Bazzite uses.

### What did you expect to happen?

The mouse cursor to stay at the same size & clarity / resolution.

### Output of `rpm-ostree status`

State: idle
Deployments:
  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:10ebb6e959a7574a56cd90451652eb3cc2ce6e406465661092875ea91d6318f3
                  Version: 41.20241104 (2024-11-04T05:00:08Z)
            SecAdvisories: 2 moderate, 1 important
                     Diff: 228 upgraded, 2 removed
          LayeredPackages: goverlay java-21-openjdk.x86_64
            LocalPackages: lact-0.5.6-0.x86_64

● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:5bb32b035ce0678ada4b86f4899c7e237887c02038e077934b214f27866de14a
                  Version: 41.20241030 (2024-10-31T01:20:26Z)
          LayeredPackages: goverlay java-21-openjdk.x86_64
            LocalPackages: lact-0.5.6-0.x86_64

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:dd21242732e272339c1e5d9cee6441f95e819223e12d53cf1430a3db517d6bd6
                  Version: 41.20241029.1 (2024-10-29T15:39:27Z)
          LayeredPackages: goverlay java-21-openjdk.x86_64
            LocalPackages: lact-0.5.6-0.x86_64

**Comment by Zeglius** [comment]
Yeah, it seems KDE is now using SVG to render their cursors, but GTK didn't implement it yet so yeah.

https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/6212

A workaround would be using a cursor theme that uses the old X11 cursor format (like Bibata Modern Classic, per example).

**Comment by Dark-Thoughts** [comment]
Hm, that causes blurry cursors with the new wiggle zoom though. Not that it is important, I just kept it enabled because I find it mildly funny. :)
How long is the average timeframe of such releases landing on (Bazzite) stable when they get merged, assuming there aren't any major hiccups?
